launch/tier4_localization_launch/launch/util/util.launch.xml

```diff
@@ -12,7 +13,7 @@
   <arg name="use_intra_process" default="true" description="use ROS2 component container communication"/>
 
   <!-- pose_initializer -->
-  <group>
+  <group if="$(eval &quot;'$(var pose_estimator_mode)'=='lidar'&quot;)">
```

Comment:
If gnss mode, we don't use pose_initializer?
We will have to check if Autoware works properly without pose_initializer.

Reply:
I didn't see the need to use pose_initializer when in gnss mode, is it necessary?
This node is ndt's intializer, right?

I have confirmed that the logging_simulator outputs control topics and that emergencies do not occur, but what else should I check?

Reply:
> This node is ndt's intializer, right?

No. This node is the "localization's" pose initializer.
If you do not use NDT, you can turn it off by `ndt_enabled` (but I've never turn it off).
https://github.com/autowarefoundation/autoware.universe/blob/main/localization/pose_initializer/docs/pose_initializer.md

Reply:
The pose_initializer is an important node in supporting the AD API, so it is likely that some functions will not work without it.

Reply:
I think we should discuss to be able to use pose_initializer in gnss mode as well.

Reply:
ok. I will take another look at the contents of pose_initializer and consider.

Reply:
Let me know if you need advice.
I think there is a possibility that we should modify pose_initializer to merge your PR.

Reply:
PR created.
https://github.com/autowarefoundation/autoware.universe/pull/2904